doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -600,7 +603,7 @@ resemble ML-like definitions of polymorphic functions.
 Abbreviations
 ~~~~~~~~~~~~~
 
-.. tacn:: set @ident {? : @term } := {? @occ_switch } @term
+.. tacn:: set @ident @ssrsetfwd {? @ssrclauses }
```

Comment:
I think we should inline `ssrsetfwd` since it is just used for `set`
```
.. tacn:: set @ident {? : @term } := {| %{ @clear_switch %} @cpattern | @lcpattern } {? @ssrclauses }
```

Reply:
Splicing a non-terminal throughout the grammar is just:

```
SPLICE: [
| ssrsetfwd
]
```

The tool updates the `tacn` automatically, no need for you to write it out